src/renderer/components/queryResult/queryVisualization/ChartRenderer.tsx

```
@@ -0,0 +1,286 @@
+              dataKey={primaryYAxisCol}
+              nameKey={xAxisCol}
+              cx="50%"
+              cy="50%"
+              outerRadius={130}
+              stroke={sliceStroke}
+              strokeWidth={2}
+              label={renderPieLabel}
+            >
+              {validPieData!.map((entry, index) => (
+                <Cell
+                  key={`cell-${index}`}
```

Biome (2.5.0)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)